robottelo/hosts.py

```diff
@@ -1374,3 +1374,20 @@ def register_to_dogfood(self):
             raise SatelliteHostError(
                 f'Error during registration, command output: {cmd_result.stdout}'
             )
+
+    def install_cockpit(self):
+        cmd_result = self.execute(
+            'foreman-installer --enable-foreman-plugin-remote-execution-cockpit', timeout='30m'
```

Comment:
```suggestion
            'satellite-installer --enable-foreman-plugin-remote-execution-cockpit', timeout='30m'
```

Reply:
it doesn't matter it is just a symlink

Reply:
In this context it does not matter much as you are not testing the installer itself.

robottelo]$ grep -r satellite-installer tests/ | wc -l
36
robottelo]$ grep -r foreman-installer tests/ | wc -l
12

Reply:
we can rename all of them, but i think it's not important, IMO it will only slow us in future development, because there will one more thing to check during the pr's, but sure we can create an issue and rename all of them later

Reply:
Regardless of everything `satellite-installer` is more accurate here and as a downstream testing. The same is suggested in documentation https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/managing_hosts/host_management_and_monitoring_using_red_hat_web_console#integrating_satellite_with_red_hat_web_console .

So use `satellite-installer`